conversion.md

```diff
@@ -0,0 +1,89 @@
+2. If both `Config.Entrypoint` and `Config.Cmd` are specified, the converter MUST append the value of `Config.Cmd` to the value of `Config.Entrypoint` and set `process.args` to that combined value.
+3. See the section on [annotations](#annotations) for caveats.
+
+## Parsed Fields
+
+Certain image configuration fields have a counterpart that must first be translated.
+A compliant configuration converter SHOULD parse all of these fields and set the corresponding fields in the generated runtime configuration:
+
+| Image Field         | Runtime Field    | Notes |
+| ------------------- | ---------------  | ----- |
+| `Config.User`       | `process.user.*` |       |
+
+### `Config.User`
+
+When converting the value of `Config.User` a converter SHOULD resolve the user information using a method appropriate for the container's context.
+For Linux based systems this would involve parsing `/etc/passwd` from the extracted container's root filesystem to determine the values of `process.user.uid` and `process.user.gid`.
+
+In addition, a converter SHOULD set the value of `process.user.additionalGids` to a value corresponding to the user described by `Config.User`.
+For Linux based systems this would require parsing `/etc/group` and determining the group memberships of the user specified in `process.user.uid`.
+
+If `Config.User` is not defined, the converter MUST choose a default user appropriate for the container's context.
+If `Config.User` does not correspond to a user in the container's context, the convert MUST return an error.
+
+## Optional Fields
+
+Certain image configuration fields are not considered to
+A compliant configuration converter SHOULD provide a way for users to extract these fields into the generated runtime configuration:
+
+| Image Field           | Runtime Field      | Notes |
+| --------------------- | ------------------ | ----- |
+| `Config.Memory`       | platform dependant | 1     |
+| `Config.MemorySwap`   | platform dependant | 1     |
+| `Config.CPUShares`    | platform dependant | 1     |
+| `Config.ExposedPorts` | none               | 2     |
+| `Config.Volumes`      | `mounts`           | 3     |
+
+1. The meaning of these fields is platform dependant, and runtimes MAY ignore these fields.
+   On Linux these image fields correspond to the runtime fields `linux.resources.memory.limit`, `linux.resources.memory.swap`, and `linux.resources.cpu.shares` respectively.
+2. The runtime configuration does not have a corresponding field for this image field.
+   Converters SHOULD use this as a hint to allow a container to provide a network service from the corresponding container-side port.
+3. The converter MUST set the `destination` of the mountpoint to the value specified in `Config.Volumes`.
+   The other fields are platform and context dependant, though the converter SHOULD NOT allow volumes to be include in newly created [layers](layer.md).
```

Comment:
“include” → “included”.

And “SHOULD NOT allow volumes to be included in the newly created layers” sounds like it may belong [here][1], and not in the conversion spec.

[1]: https://github.com/opencontainers/image-spec/blob/v1.0.0-rc3/layer.md#determining-changes